projects/portal/src/app/views/delegate/validators/validators.component.html

```diff
@@ -56,57 +59,66 @@ <h2 class="mb-0">Delegations</h2>
-      </mat-list-item>
-    </mat-list>
-    <mat-nav-list>
-      <ng-container *ngFor="let unbondingDelegation of unbondingDelegations; last as last">
-        <ng-container *ngFor="let validator of delegatedValidators">
-          <mat-list-item
-            *ngIf="unbondingDelegation?.unbond?.validator_address === validator?.operator_address"
-            (click)="onClickValidator(validator!)"
-          >
-            <mat-icon
-              matListIcon
-              [ngStyle]="{
-                color: getColorCode(unbondingDelegation?.unbond?.validator_address ?? '')
-              }"
+  <div class="overflow-x-auto mb-8">
+    <table class="table w-full">
+      <!-- head -->
+      <thead>
+        <tr>
+          <th></th>
+          <td>Validator</td>
+          <td>Date</td>
+          <td>Amount</td>
+        </tr>
+      </thead>
+      <tbody>
+        <ng-container *ngFor="let unbondingDelegation of unbondingDelegations">
+          <ng-container *ngFor="let validator of delegatedValidators">
+            <tr
+              class="hover cursor-pointer"
+              *ngIf="unbondingDelegation?.unbond?.validator_address === validator?.operator_address"
+              (click)="onClickValidator(validator!)"
             >
-              circle
-            </mat-icon>
-            <span class="w-6/12 break-all text-sm sm:text-base"
-              >{{ validator?.description?.moniker }}
-            </span>
-            <span class="w-4/12 break-all text-sm sm:text-base text-right pr-2"
-              >{{
-              unbondingDelegation?.unbond?.entries?.[0]?.completion_time | date : 'yyyy-MM-dd a hh:mm:ss z'}}
-            </span>
-            <span class="w-2/12 break-all text-sm sm:text-base text-right pr-2">{{
-              unbondingDelegation?.unbond?.entries?.[0]?.balance }}</span>
-          </mat-list-item>
+              <th>
+                <mat-icon
+                  matListIcon
+                  [ngStyle]="{
+                    color: getColorCode(unbondingDelegation?.unbond?.validator_address ?? '')
+                  }"
+                >
+                  circle
+                </mat-icon>
+              </th>
+              <td>{{ validator?.description?.moniker }}</td>
+              <td>
+                {{ unbondingDelegation?.unbond?.entries?.[0]?.completion_time | date : 'yyyy-MM-dd a hh:mm:ss z'}}
+              </td>
+              <td>
+                {{ unbondingDelegation?.unbond?.entries?.[0]?.balance ?? '0' | coinAmount }}
+              </td>
+            </tr>
+          </ng-container>
         </ng-container>
-        <mat-divider *ngIf="!last" [inset]="true"></mat-divider>
-      </ng-container>
-    </mat-nav-list>
-  </mat-card>
-</ng-template>
-<ng-template #emptyUnbonding>
-  <p>There is no unbonding delegations.</p>
+      </tbody>
+    </table>
+  </div>
 </ng-template>
+<ng-template #emptyUnbonding></ng-template>
 
 <div class="flex flex-row items-end mb-2">
```

Comment:
shouldn't be items-end, and mb-2 and mb of h2 should b eliminated in design principle

Reply:
This PR updated.
but t is present in many components. Should I remove them?

Reply:
what is t?

Reply:
so you mean this?

let's do later